zubhub_frontend/zubhub/src/views/search_results/SearchResults.jsx

```diff
@@ -173,7 +177,40 @@ function SearchResults(props) {
     next: next_page,
     loading,
   } = state;
-  const { t } = props;
+  const { t, auth } = props;
+
+  if (!auth.token) {
+    return (
+      <Container className={modalClasses.root}>
+        <Grid className={modalClasses.projectContainer}>
+          <Grid item xs={12}>
+            <Typography
+              className={classes.pageHeaderStyle}
+              variant="h3"
+              gutterBottom
+              style={{ height: '40px'}}
+            >
+              {`${t('searchResults.resultsFound')} "${getQueryParams(window.location.href).get('q')}"`}
+            </Typography>
+          </Grid>
+          {getResults(
+            getQueryParams(window.location.href).get('type'),
+            props.auth.token ? results : results[0]?.projects?.results,
```

Comment:
also check if results is not an empty array

Reply:
Handled in `getResults` function where I show the `ErrorComponent`